README.md

```diff
@@ -29,6 +29,18 @@ Settings API into reuseable classes and implements a set of simple controls.
 - Google Map Address (with GeoLocation)
 - Business Opening Hours
 
+## Installing with Composer
+
+You can install this library using Composer, by adding following lines to repositories;
+```
+{
+	 "url": "https://github.com/l3pp4rd/DoctrineExtensions.git",
+	 "type": "git"
+}
+```
```

Comment:
Where does this go in a project and why is it necessary? I admit I'm not terribly familiar with Composer, but in all the times I've used it I only needed a `composer.json` file which referenced the dependencies, then I could just run `composer install` without any problem.